Makefile

```diff
@@ -0,0 +1,36 @@
+Command := $(firstword $(MAKECMDGOALS))
+Arguments := $(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+
+default: show 
+
+bhyveImage:
+	@echo "nix building $(Arguments) bhyve Image"
+	nix build ".#nixosConfigurations.$(Arguments).config.system.build.bhyveImage"
+
+qemuVm:
+	@echo "nix building $(Arguments) qemu VM"
+	nix build ".#nixosConfigurations.$(Arguments).config.system.build.vm"
+
+all:	bhyveImage qemuvm
+        	
+
+show:
```

Comment:
Nice I like this addition! Actually caused me to spin out a follow up issue (https://github.com/socallinuxexpo/scale-network/issues/600) since it looks like I have to do something a little more cleaver for the package scope for scaleTests in the flake.nix